libnavui-maps/api/current.txt

```diff
@@ -415,6 +416,15 @@ package com.mapbox.navigation.ui.maps.guidance.junction.api {
   public static final class MapboxJunctionApi.Companion {
   }
 
+  public final class MapboxRasterToBitmapParser implements com.mapbox.navigation.ui.maps.guidance.junction.api.RasterToBitmapParser {
+    ctor public MapboxRasterToBitmapParser();
+    method public com.mapbox.navigation.ui.base.model.Expected<android.graphics.Bitmap,java.lang.String> parse(byte[] raster);
+  }
+
+  public interface RasterToBitmapParser {
+    method public com.mapbox.navigation.ui.base.model.Expected<android.graphics.Bitmap,java.lang.String> parse(byte[] raster);
+  }
+
```

Comment:
Let's not expose these interfaces/implementations. The operation is trivial enough that I doubt it will ever need an alternative implementation.

Reply:
I agree. But shouldn't we leave it for consistency sake given that we do the same for Signboards.

Reply:
I am gonna remove the interface, but I am gonna keep the implementation and make it internal. It's good to handle the logic separately outside the `Processor`.

Reply:
:+1: